src/core/Akka.Tests.Shared.Internals/Helpers/FSharpDelegateHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="FSharpDelegateHelper.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using Microsoft.FSharp.Core;
+
+namespace Akka.Tests.Shared.Internals.Helpers
+{
+    /// <summary>
+    /// Maps F# methods to C# delegates
+    /// </summary>
+    public static class FsharpDelegateHelper
```

Comment:
Helper class I've used in a ton of other projects for converting `Func<>` into F# first-order functions... Useful tool when working with FsCheck and you want to use something like `Gen.Map2`